functions/_fzf_search_history.fish

```diff
@@ -2,14 +2,10 @@ function _fzf_search_history --description "Search command history. Replace the
     # history merge incorporates history changes from other fish sessions
     builtin history merge
 
-    # Make sure that fzf uses fish so we can run fish_indent.
-    # See similar comment in _fzf_search_variables.fish.
-    set --local --export SHELL (command --search fish)
```

Comment:
this was also unnecessary b/c fish_indent is a binary that is most likely available in other shells